### ISSUE TITLE
Copy-Item does not transfer blob data for media items

### ISSUE THREAD

**Issue by michaellwest** [SitecorePowerShell/Console#782]
Discovered that the following command will not transfer blob data.

```powershell
Copy-Item -Path "master:/media library/Images/SPE" -Destination "web:/media library/Images/" -TransferOptions 0 -Recurse
orce
```

Result:
![image](https://cloud.githubusercontent.com/assets/933163/21754314/6c721b98-d5c3-11e6-9978-30442894ba96.png)

**Comment by michaellwest** [comment]
@MichaelBosch Thank you for bringing this issue to our attention. Turns out the functionality was completely missing from the `Copy-Item` command. I've now been able to address it.

**Comment by michaellwest** [comment]
Fixed.

**Comment by MichaelBosch** [comment]
Thanks for the quick feedback and fix!

**Comment by AdamNaj** [comment]
What's the verdict @michaellwest? Are we compatible with all Sitecore versions now?

**Comment by michaellwest** [comment]
Yes. I'm happy to announce that the `Copy-Item` unit test made the verification pain free.

Validated on SC 7.0 rev 140408 
1. Installed SPE 4.3 (had to disable Cognifide.PowerShell.Events.config)
2. Ran unit test which failed on the blob check
3. Copied debug binary for 7.0
4. Ran unit test which passed on the blob check

**Comment by AdamNaj** [comment]
I would like to avoid bugs being part of the release-highlight. Changing the qualification to improvement.

**Comment by dthunziker** [comment]
Did this improvement ever made it into a release? I ran into the same issue, but worked around it by transferring the blob separately:

```
$sourceStream = $sourceImageField.MediaItem.Fields["Blob"].GetBlobStream()
$newItem.Editing.BeginEdit()
$newItem.Fields["Blob"].SetBlobStream($sourceStream )
$newItem.Editing.EndEdit()
```

**Comment by michaellwest** [comment]
@dthunziker it certainly should be in the releases following this commit. What version did you write this script for?

**Comment by dthunziker** [comment]
Sitecore PowerShell Extensions 4.3.0.17491
Sitecore 8.2
Windows PowerShell 5.1

**Comment by michaellwest** [comment]
it sure does look like you just missed it. We released 5.1 and working on 6.0!
